conf/standalone.conf

```diff
@@ -752,6 +752,16 @@ managedLedgerAddEntryTimeoutSeconds=0
 # Of course, use a smaller value may degrade consumption throughput. Default is 10ms.
 managedLedgerNewEntriesCheckDelayInMillis=10
 
+# Minimum cursors that must be in backlog state to cache and reuse the read entries.
+# (Default =0 to disable backlog reach cache)
+managedLedgerMinimumBacklogCursorsForCaching=0
+
+# Minimum backlog entries for any cursor before start caching reads.
+managedLedgerMinimumBacklogEntriesForCaching=100
+
+# Maximum backlog entry difference to prevent caching entries that can't be reused.
+managedLedgerMaxBacklogBetweenCursorsForCaching=10000
```

Comment:
@rdhabalia - was it intentional for these two settings to differ from the settings in the `ManagedLedgerConfig` class?